WordPress/wordpress_migration_linux_appservices.md

```diff
@@ -1,134 +1,125 @@
-## 2. Manual Migration Process
-The prerequisite is that the WordPress on Linux Azure App Service must have been created with an appropriate hosting plan from here: [WordPress on Linux App Service](./https://portal.azure.com/#create/WordPress.WordPress).
+1. Create a new Wordpress app using our [WordPress on Linux App Service template](https://aka.ms/linux-wordpress)
 
+1. Open an SSH session using **WebSSH** from the Azure portal.
+![Web SSH](./media/post_startup_script_1.png)
 
-### Export the data at source site: 
-1. Download the **wp-content** folder from the source site. You can use popular FTP tools like [FileZilla](https://filezilla-project.org/download.php?type=client) to connect to the web server and download the content. Depending on the size of your content, it could take sometime. Meanwhile, you can go to the next step to download a copy of your database. 
+1. Delete the existing content of **/home/site/wwwroot/wp-content** folder using the following command.
 
-2. Export the contents of the source database into an SQL file. You can perform this task either using MySQL client tools like HeidiSQL, MySQL workbench, PhpMyAdmin or through command line interface. You can download the MySQL Workbench from [here](https://dev.mysql.com/downloads/workbench/) or install phpmyadmin using these [steps](https://docs.phpmyadmin.net/en/latest/setup.html). For more information on exporting the database, please refer to the following [documentation](https://dev.mysql.com/doc/workbench/en/wb-admin-export-import-management.html).
+   ```bash
+   rm -rf /home/site/wwwroot/wp-content/* 
+   ```
 
-### Import the data at destination site: 
-1. Go to SCM site of your WordPress App Service using the following URL.
-    ````
-    https://<sitename>.scm.azurewebsites.net/newui 
-    ````
-2. Open Bash Shell from the menu shown on the top.
-3. Delete the existing content of **/home/site/wwwroot/wp-content** folder using the following command.
-    ````
-    rm -rf /home/site/wwwroot/wp-content/* 
-    ````
-4. Now upload the new contents of **wp-content** folder using the File Manager. Click on the label that says '**Drag a File/Folder here to upload, or click to select one**'. Please note that if you are not able to upload everything at once, then you can try dividing your upload into multiple smaller ones.
+1. Upload the new contents of **wp-content** folder using the File Manager. Click on the label that says '**Drag a File/Folder here to upload, or click to select one**'. Please note that if you are not able to upload everything at once, then you can try dividing your upload into multiple smaller ones.
 
-5. You can either point your WordPress to [use an existing MySQL database](./using_an_existing_mysql_database.md), or use the steps below to migrate the content to the new database server created by the WordPress on Linux App Services offering.
+   <!TODO: we should prob recomend doing this over FTP, it's more reliable than web upload through SCM site?>
```

Comment:
@rabollin see my comment in the TODO

Reply:
@btardif Will review and raise a separate PR for this